gwpy/table/filter.py

```diff
@@ -205,12 +205,14 @@ def _flatten(container):
 def is_filter_tuple(tup):
     """Return whether a `tuple` matches the format for a column filter
     """
-    return (
-        isinstance(tup, (tuple, list))
-        and len(tup) == 3
-        and isinstance(tup[0], str)
-        and callable(tup[1])
-    )
+    try:
+        names, func, args = tup
+        return (
+            (isinstance(names, str) or all(isinstance(x, str) for x in names))
+            and callable(func)
+        )
```

Comment:
Seems like the logic conditionals have changed here. Was there something incorrect about the original and/or what is the motivation for the updated version?

Reply:
The latter, the first two conditions are now taken care of by the tuple unpacking, and the rest is just to support the first element of the tuple being a list of names.

Reply:
Sounds good, thanks for confirming